pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/plugin/PythonWebApplicationPlugin.java

```diff
@@ -33,11 +34,15 @@
 
     @Override
     public void applyTo(final Project project) {
-
         project.getPlugins().apply(PythonContainerPlugin.class);
 
         final DeployableExtension deployableExtension = ExtensionUtils.maybeCreateDeployableExtension(project);
 
+        // 2019-04-11(warsaw): FIXME: For now, we're still hard coding pex
+        // for the gunicorn file.  Make sure the `pex` dependency is
+        // installed.
+        new PexExtension(project).addDependencies(project);
```

Comment:
I think we don't, at least in my own testing on several products.  We can verify that it's the case with (internal) testing, but this is a pure hack to get `pex` installed at the right time for the hard coded use of pex for the gunicorn file.  It will all go away soon.